src/main/java/wooteco/subway/line/dao/LineDao.java

```diff
@@ -1,77 +1,92 @@
-
 @Repository
 public class LineDao {
+
+    private static RowMapper<Line> ROW_MAPPER() {
+        return (rs, rowNum) -> {
+            Long foundId = rs.getLong("id");
+            final String color = rs.getString("color");
+            final String name = rs.getString("name");
+            return new Line(foundId, name, color);
+        };
+    }
+
     private JdbcTemplate jdbcTemplate;
     private SimpleJdbcInsert insertAction;
 
     public LineDao(JdbcTemplate jdbcTemplate, DataSource dataSource) {
         this.jdbcTemplate = jdbcTemplate;
         this.insertAction = new SimpleJdbcInsert(dataSource)
-                .withTableName("line")
-                .usingGeneratedKeyColumns("id");
+            .withTableName("LINE")
+            .usingGeneratedKeyColumns("id");
     }
 
     public Line insert(Line line) {
         Map<String, Object> params = new HashMap<>();
         params.put("id", line.getId());
         params.put("name", line.getName());
         params.put("color", line.getColor());
+        params.put("extra_fare", line.getExtraFare());
 
         Long lineId = insertAction.executeAndReturnKey(params).longValue();
-        return new Line(lineId, line.getName(), line.getColor());
+        return new Line(lineId, line.getName(), line.getColor(), line.getExtraFare());
     }
 
     public Line findById(Long id) {
         String sql = "select L.id as line_id, L.name as line_name, L.color as line_color, " +
-                "S.id as section_id, S.distance as section_distance, " +
-                "UST.id as up_station_id, UST.name as up_station_name, " +
-                "DST.id as down_station_id, DST.name as down_station_name " +
-                "from LINE L \n" +
-                "left outer join SECTION S on L.id = S.line_id " +
-                "left outer join STATION UST on S.up_station_id = UST.id " +
-                "left outer join STATION DST on S.down_station_id = DST.id " +
-                "WHERE L.id = ?";
+            "S.id as section_id, S.distance as section_distance, " +
+            "UST.id as up_station_id, UST.name as up_station_name, " +
+            "DST.id as down_station_id, DST.name as down_station_name, " +
+            "L.extra_fare as extra_fare " +
+            "from LINE L \n" +
+            "left outer join SECTION S on L.id = S.line_id " +
+            "left outer join STATION UST on S.up_station_id = UST.id " +
+            "left outer join STATION DST on S.down_station_id = DST.id " +
+            "WHERE L.id = ?";
 
         List<Map<String, Object>> result = jdbcTemplate.queryForList(sql, new Object[]{id});
         return mapLine(result);
     }
 
     public void update(Line newLine) {
         String sql = "update LINE set name = ?, color = ? where id = ?";
-        jdbcTemplate.update(sql, new Object[]{newLine.getName(), newLine.getColor(), newLine.getId()});
+        jdbcTemplate
+            .update(sql, new Object[]{newLine.getName(), newLine.getColor(), newLine.getId()});
```

Comment:
```suggestion
            .update(sql, newLine.getName(), newLine.getColor(), newLine.getId());
```

이미 아실 수 있지만..! 배열로 감싸지 않고 입력하여도 가능하다는 사실!